keyboards/nightly_boards/n87/keymaps/via/rules.mk

```
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
+LINK_TIME_OPTIMIZATION_ENABLE = yes
+BOOTMAGIC_ENABLE = lite
```

Please use the short form of LTO here.
Also, if you enable VIA, it enables bootmagic lite. And you already have it enabled at the keyboard level. 
```suggestion
LTO_ENABLE = yes
```

---

https://github.com/qmk/qmk_firmware/pull/9885/commits/85db5dc34a97bb83d74a031fb505ce8fcfc2cbba